bindings/masm/KSExample_x64/KSExample_x64.Asm

```diff
@@ -0,0 +1,210 @@
+		
+        Invoke GetSysColorBrush, COLOR_WINDOW
+        mov hWhiteBrush, eax		
+		
+		Invoke GetDlgItem, hWin, IDC_TxtAsmToEncode
+        mov hTxtAsmToEncode, rax
+
+        Invoke GetDlgItem, hWin, IDC_TxtEncodedBytes
+        mov hTxtEncodedBytes, rax
+        
+        Invoke SetWindowText, hTxtAsmToEncode, Addr szCODEInitial
+		
+	.ELSEIF eax == WM_COMMAND
+	    mov rax, wParam
+		.IF rax == IDM_FILE_EXIT || rax == IDC_BtnExit
+			Invoke SendMessage, hWin, WM_CLOSE, 0, 0
+			
+		.ELSEIF rax == IDM_HELP_ABOUT
+			Invoke ShellAbout, hWin, Addr AppName, Addr AboutMsg, NULL
+		
+		.ELSEIF rax == IDC_BtnEncode
+		    Invoke DoEncode, hWin
+			
+		.ENDIF
+
+    .ELSEIF eax == WM_CTLCOLORSTATIC
+        mov eax, hWhiteBrush
+        ret
+
+	.ELSEIF eax == WM_CLOSE
+		Invoke DestroyWindow, hWin
+		
+	.ELSEIF eax == WM_DESTROY
+		Invoke PostQuitMessage, NULL
+		
+	.ELSE
+		invoke DefWindowProc, rcx, edx, r8, r9
+		ret
+	.ENDIF
+	xor rax, rax
+	ret
+WndProc endp
+
+
+;-------------------------------------------------------------------------------------
+; DoEncode - Keystone encode assembler in text box to bytes
+;-------------------------------------------------------------------------------------
+DoEncode PROC FRAME USES RBX hWin:QWORD
+    LOCAL i:QWORD
+    
+    ; Fetch text user typed (or default code string)
+    Invoke GetWindowText, hTxtAsmToEncode, Addr szCODEBuffer, SIZEOF szCODEBuffer
+    .IF rax == 0
+	    Invoke MessageBox, 0, Addr szKSCodeEmpty, Addr szKSError, MB_OK
+	    ret
+    .ENDIF
+    
+    ; Reset some stuff
+    Invoke RtlZeroMemory, Addr szFinalOutput, SIZEOF szFinalOutput
+    Invoke SetWindowText, hTxtEncodedBytes, Addr szNull
+
+    ; Keystone stuff
+	Invoke ks_version, KS_VERSION_MAJOR, KS_VERSION_MINOR
+	; rax contains version
+	
+	Invoke ks_arch_supported, KS_ARCH_X86
+	; rax contains TRUE if arch is supported by keystone library
+	
+	Invoke ks_open, KS_ARCH_X86, KS_MODE_32, Addr ks_engine
+	mov ks_err, rax
+	.IF rax != KS_ERR_OK
+	    Invoke MessageBox, 0, Addr szKSOpenFail, Addr szKSError, MB_OK
+	    ret 
+	.ENDIF
+
+    ; Encode assembler code to bytes
+	Invoke ks_asm, ks_engine, Addr szCODEBuffer, 0, Addr Encode, Addr EncodeSize, Addr Count
+	mov ks_err, rax
+	.IF rax != KS_ERR_OK
+	    Invoke MessageBox, 0, Addr szKSAsmFail, Addr szKSError, MB_OK
+	    ret
+	.ELSE
+	    
+	    ; loop through encoding, fetch each byte and add to our final output string
+	    mov i, 0
+	    mov rax, 0
+	    .WHILE rax < EncodeSize
+	    
+	        mov rbx, Encode
+	        add rbx, i
+	        movzx rax, byte ptr [rbx]
+	        
+	        Invoke wsprintf, Addr szOutput, Addr szFmt, rax
+            Invoke lstrcat, Addr szFinalOutput, Addr szOutput
+            
+	        inc i
+	        mov rax, i
+	    .ENDW
+	.ENDIF
+	
+	; Clean up stuff
+	Invoke ks_free, Encode
+	Invoke ks_close, ks_engine
+
+    ; Show encoded bytes to user
+    Invoke SetWindowText, hTxtEncodedBytes, Addr szFinalOutput
+
+    ret
+
+DoEncode endp
+
+
```

Comment:
can you remove all these blank lines (and in other files)?

Reply:
Ok think I got them now, I removed the blank lines at the end of the text files and converted tabs to spaces as well